Dockerfile

```diff
@@ -0,0 +1,67 @@
+# This is a multi-stage build which requires Docker 17.05 or higher
+FROM python:3.7-alpine as molecule-builder
+
+WORKDIR /usr/src/molecule
+
+ENV PACKAGES="\
+    gcc \
+    git \
+    libffi-dev \
+    make \
+    musl-dev \
+    openssl-dev \
+    "
+RUN \
+    apk update && \
```

Comment:
Is this necessary with `apk --no-cache`?

Reply:
Yes, it updates the list of packages

Reply:
What for? The next command ignores the cache anyways.

Reply:
https://stackoverflow.com/a/49119046

Reply:
Ah, ok